app/src/main/java/io/github/zadam/triliumsender/LoginActivity.kt

```diff
@@ -37,7 +37,31 @@ class LoginActivity : AppCompatActivity() {
             false
         })
 
-        loginButton.setOnClickListener { attemptLogin() }
+        trustModeRadio.setOnCheckedChangeListener(
+                RadioGroup.OnCheckedChangeListener { radio, _ ->
+                    val checked = radio.checkedRadioButtonId
+
+                    if (checked == useDefaultTrust.id) {
+                        trustAllWarning.visibility = View.GONE
+                        customCertInput.visibility = View.GONE
+                    } else if (checked == useCustomCert.id) {
+                        trustAllWarning.visibility = View.GONE
+                        customCertInput.visibility = View.VISIBLE
+                    } else {
+                        trustAllWarning.visibility = View.VISIBLE
+                        customCertInput.visibility = View.GONE
+                    }
+                }
+        )
```

Comment:
Perhaps I just can't see it, but is the cert setting saved somewhere? I can see that the login will be attempted with the chosen cert setting, but then when I close the app and want to use it to send some text note to trilium server, how will the app know which cert settings has been chosen before?